### ISSUE TITLE
User Authentication (Login)

### ISSUE THREAD

**Issue by pooranjoyb** [sristy17/MindDrive#2]
- Use the collection.json from Release and import it to your Postman.
- Use the predefined routes to test the APIs
- Integrate the Login feature

**Comment by theCodeCrusaderX** [comment]
please assign this issue to me

**Comment by pooranjoyb** [comment]
Go ahead @theCodeCrusaderX

**Comment by AratiAmbekar** [comment]
please assign me this issue.

**Comment by pooranjoyb** [comment]
> please assign me this issue.

Hey @AratiAmbekar as of now, Auth is being done by @theCodeCrusaderX , you can work on the conditional rendering of the profile once that part is done.

will assign you that.

**Comment by theCodeCrusaderX** [comment]
![login form](https://github.com/user-attachments/assets/76e5b55b-f835-4fea-9eb3-d5a2b95667d3)


i have checked from my end all works fine 

user signIn done

**Comment by theCodeCrusaderX** [comment]
one more thing i know this is not related to this repo but as i am new to hactoberfest .

i want to ask that why my progress in hactoberfest profile is still showing 0/4 [1 waiting] .

and one more thing will i get a separate pull request fom signin to??
